src/harness/sas.cfg

```diff
@@ -1,3 +1,6 @@
 [SasConfig]
 BaseUrl: localhost:9000
 Version: v1.2
+
+[SasTestHarnessConfig]
```

Comment:
The section [SasConfig]  contains the parameters for the SAS UUT. We wanted this to be kept separate for SAS UUT and SAS Test Harness to avoid any unintended use. I think you would like to have the version field fixed for both UUT and TH as there are no tests right now that use different version. We have removed this now and used the same version field in [SasConfig]. The sas.cfg file is removed from the PR.

Reply:
This doesn't appear to have happened. Please check.